### PR TITLE
Update from template repo: PRs 15, 24, 26

### PR DESCRIPTION
**Pull request by rocketnova**
## Ticket

N/A

- https://github.com/navapbc/template-application-nextjs/pull/15
- https://github.com/navapbc/template-application-nextjs/pull/24
- https://github.com/navapbc/template-application-nextjs/pull/26
- https://github.com/navapbc/template-application-nextjs/pull/29
  - Note: These features were already enabled in this repo, so this commit is just bringing in stylistic updates to make the two repos identical.
- https://github.com/navapbc/template-application-nextjs/pull/37

This will also close:

- #29 
- #26 
- #25 
- #12 
- #11

### CODE REVIEW

File: app/README.md [hunk 1]
```diff
@@ -56,8 +56,6 @@ Note: make sure TypeScript and Javascript Language Features are enabled in VS Co
   - Disallows usage of `any` type. The usage of `any` defeats the purpose of typescript. Consider using `unknown` type instead instead.
 - "react/resct-in-jsx-scope": "off"
   - suppress errors for missing 'import React' in files because NextJS does this for us.
-- "space-before-function-paren": ["error", "never"]
-  - suppresses errors for lack of space before function parenthesis to allow for `function()`
```
Comment (aplybeah) [inline app/README.md:60]: Why was this removed? Is it no longer needed?

**Review by aplybeah — APPROVED** [review]
🎉

File: app/README.md (hunk 1 above)
Comment (rocketnova) [reply app/README.md:60]: I went with the opposite approach:

> disabling style lint rules and using whatever the code formatter's default is

See [discussion](https://github.com/navapbc/template-application-nextjs/pull/24#discussion_r920556414).
